src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -346,32 +318,17 @@ public static Score parseScore(String score) throws ParseException {
         return new Score(trimmedScore);
     }
 
-    /**
-     * Parses a {@code String lessonDate} into a {@code Attendance} formatted {@code String}.
-     * Leading and trailing whitespaces will be trimmed.
-     *
-     * @throws ParseException if the given {@code lessonDate} is invalid.
-     */
-    public static String parseAttendanceDate(String lessonDate) throws ParseException {
-        requireNonNull(lessonDate);
-        String trimmedLessonDate = lessonDate.trim();
-        if (!Attendance.isValidDate(trimmedLessonDate)) {
-            throw new ParseException(Attendance.DATE_CONSTRAINTS);
-        }
-        return trimmedLessonDate;
-    }
-
     /**
      * Parses a {@code String attendanceStatus} into a {@code Attendance} formatted {@code String}.
      * @throws ParseException if the given {@code attendanceStatus} is invalid.
      */
```

Comment:
Minor change, could you update the javadoc as well

Reply:
Good catch!